sgx/Cargo.toml

```diff
@@ -0,0 +1,4 @@
+[workspace]
+
+[workspace.package]
+rust-version = "1.68.0"
```

Comment:
just curious, what's the reason for this change? why do we need a new workspace for this?

Reply:
It allows all of the sub-packages to access this value. That way, when we update the MSRV again, we don't have to change ~12 values; we only have to change this one value.

Reply:
Ended up implementing your suggestion, see comment below.